systemc/NoSysC_test.cpp

```diff
@@ -0,0 +1,149 @@
+#include <iostream>
+#include <memory>
+#include <vector>
+#include <gtest/gtest.h>
+#include <glog/logging.h>
+#include "model_rsa.h"
+#include "verilog/dtype/vint.h"
+#include "modellib/cport.h"
+#include "modellib/cvalidready.h"
+#include "modellib/cmodule.h"
+
+using namespace verilog;
+using namespace std;
+
+struct TwoPower_NSC : public Module<TwoPower_NSC> {
+	vrslave<TwoPowerIn> data_in;
+	vrmaster<TwoPowerOut> data_out;
+
+	ExtendKeyType round_result_r, modulus_r;
+	IntType power_r;
+	bool running_r, running_up_w, running_down_w;
+
+	TwoPower_NSC() {
```

Comment:
I think there is a `Reset` slot, should we implement the slot to set the state variable like `running_r/up_w/down_w`? It is a little hard to know the initial state of those variable, and thus hard to understand the behavior of Comb0, Seq0.